compiler/extccomp.nim

```diff
@@ -14,8 +14,52 @@
 
 import
   ropes, os, strutils, osproc, platform, condsyms, options, msgs,
-  securehash, streams
+  streams
 
+when defined(useMetroHash64):
+  import metrohash
```

Comment:
Where is this file? I don't see it in "Files changed".

Reply:
Hmm, it's very strange, in my local branch this file present.
I tried to fix it.